test/mongodb.test.js

```diff
@@ -82,10 +82,19 @@ describe('mongodb connector', function() {
       power: { type: String, index: true, unique: true },
       address: { type: String, required: false, index: { mongodb: { unique: false, sparse: true }}},
       description: { type: String, required: false },
-      geometry: { type: Object, required: false, index: { mongodb: { kind: '2dsphere' }}},
```

Comment:
I could, but that means creating even more data (which you wanted to avoid earlier) for little or no gain: the key is still created and properly checked so the test doesn't really change.

Even better: the test is called something like "creating complex indexes", so with a name like that it's even more complex and makes use of a different way of creating an index which isn't really tested elsewhere.